_episodes/01-connecting.md

```diff
@@ -140,14 +226,27 @@ example computer, we will use SSH (if you are using PuTTY, see above).
 
 SSH allows us to connect to UNIX computers remotely, and use them as if they
 were our own. The general syntax of the connection command follows the format
-`ssh yourUsername@some.computer.address` Let's attempt to connect to the HPC
-system now:
+`ssh yourUsername@some.computer.address` and 
+`ssh -i ~/.ssh/key_for_remote_computer yourUsername@remote.computer.address` 
+when using SSH keys.  Let's attempt to connect to the HPC system now:
 
 ```
 ssh yourUsername@{{ site.workshop_host_login }}
```

Comment:
Delete the keyless SSH command: we don't want password-based authentication.

Reply:
Thanks for the fix

Reply:
A fair point. This could be handled using snippets, or with a conditional to switch between password, RSA, and EdDSA for SSH. I'll open a follow-up issue.